.github/scripts/translate-strings.py

```
@@ -0,0 +1,229 @@
+        }
+        
+        # Process each language
+        for lang in valid_languages:
+            print(f"🔄 Translating to {lang}...")
+            
+            target_lang = LANGUAGE_MAPPING[lang]
+            texts_to_translate = list(strings.values())
+            
+            # Translate all strings
+            translated_texts = self.client.translate_multiple(texts_to_translate, target_lang)
+            translated_strings = dict(zip(strings.keys(), translated_texts))
```

Ruff (0.14.14)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)